vue.js

```diff
@@ -16,6 +16,14 @@ module.exports = {
 
     // Priority C: Recommended (Minimizing Arbitrary Choices and Cognitive Overhead)
     'vue/no-v-html': 'off', // override for translation
+    'vue-a11y/label-has-for': [
+      2,
+      {
+        required: {
+          some: ['nesting', 'id'], // Only require nesting or id
```

Comment:
Something like:

```suggestion
          // Only require one of nesting _or_ id.
          some: ['nesting', 'id'],
```

Though it's not big deal to even have the comment if you don't want.